app/assets/stylesheets/site/views/card.overrides

```diff
@@ -1,3 +1,13 @@
 /*******************************
     User Variable Overrides
 *******************************/
+
+.ui.card > .content > .description {
+    margin: 1.5em 0 !important;
+}
+
+.ui.card > .content.cardAccent {
+    background-color: #e2e2e2;
```

Comment:
This makes parts of certain cards also grey. My suggestion is to apply this style on `Cardcontent extra={true}` and keep this consistently for buttons.

Reply:
And perhaps put these colours in a global variable file as well.